.github/workflows/test.yml

```diff
@@ -86,26 +85,8 @@ jobs:
   ## Work through all of the variations of the different features, only on linux to save concurrent resources
   exhaustive-features-matrix:
     name: exhaustive
-    # Let's wait for the all-features cache
```

Comment:
Furthermore, there aren't any workflow steps that store or retrieve a cache, so this is just adding sequencing